build-npm-packages.sh

```diff
@@ -4,6 +4,7 @@ set -e
 
 yarn
 yarn workspace @joystream/types build
+yarn workspace @joystream/content-metadata-protobuf build:ts
 yarn workspace query-node-root build
 # For now cli build is broken.. proceed anyway
 yarn workspace @joystream/cli build || :
```

Comment:
Allowing CLI build to fail is no longer needed. The commentary and `|| :` can be removed.

Reply:
Done in https://github.com/Joystream/joystream/pull/2334/commits/232282f617e51f8cfde57b80eca23bcb8ee15436